fix time schedule for Matsue 1st.

emorima opened the pull request:
以下の内容でタイムスケジュールを更新しました。

<img width="608" height="616" alt="image" src="https://github.com/user-attachments/assets/b702dda5-5dcd-494a-a370-cf204b77d96b" />

emorima committed:
fix time schedule for Matsue 1st.

ksbmyk commented on events/2025-11-08-matsue_1st.markdown:
```diff
@@ -33,11 +33,52 @@ sns_message: 2025/11/08 mruby Girls Matsue 1st 開催!!!
+          <div class="plan-time">14:00  ワークショップ</div>
+          <div class="plan-description">
+            PicoRubyといろいろなセンサーを組み合わせて、<br/>
+            自分だけのガジェットを作ってみよう!
```
nits
「作ってみましょう！」のほうが他の言い回しと揃っていいですかね？

emorima replied:
確かに、言い回しが統一されていなかったので、修正しました！

ksbmyk commented on events/2025-11-08-matsue_1st.markdown:
```diff
@@ -18,7 +18,7 @@ sns_message: 2025/11/08 mruby Girls Matsue 1st 開催!!!
       <ul>
         <li>対象者：プログラミング経験がある女性（経験年数は問いません。）</li>
-        <li>必要なもの：ノートパソコン、ノートパソコンとマイコンをつなぐType-Cケーブル、好奇心</li>
+        <li>必要なもの：ノートパソコン、ノートパソコンとAtom MatrixをつなぐType-Cケーブル、好奇心</li>
```
相談ですが
Atom Matrix（やセンサー）はこちらで用意すること書いておいた方がいいですかね？（Atom Matrixが必要と勘違いさせてしまっては申し訳ないなと思って）

もしくは、データ通信に対応したType-Cケーブルもこちらで用意して「ノートパソコンと好奇心」だけで来てもらった方がシンプルですかね？

emorima replied:
Type-Cケーブル、データ転送用が必要ですが判断難しいので、たしかにこちらで準備するほうがよさそうなので、
Type-Cケーブルは削除しました！

ksbmyk reviewed (commented):
（一緒に内容詰めたので）概ねよさそうなんですが相談コメントしてます 🙏

emorima committed:
言い回しの統一: 「作ってみよう!」を「作ってみましょう!」

レビュー指摘にもあったように、
言い回しが統一されていなかったので、統一するために
「作ってみましょう!」に修正しました。

emorima committed:
必要なものから、Type-Cケーブルを削除

Type-Cケーブルはデータ転送用のケーブルである必要があり、
持ってきてもらったものが使用できない場合もある。

購入して持ってきてもらっても使用できないことを考えると、
運営側で準備して使うようにするほうがよいと判断しました。

ksbmyk reviewed (approved):
ありがとうございます！LGTM！
